src/Altinn.Platform/Altinn.Platform.Events/Events/Repository/EventsRepository.cs

```diff
@@ -0,0 +1,90 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Threading.Tasks;
+using Altinn.Platform.Events.Configuration;
+using Altinn.Platform.Events.Models;
+using Microsoft.Azure.Documents;
+using Microsoft.Azure.Documents.Client;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+
+namespace Altinn.Platform.Events.Repository
+{
+    /// <summary>
+    /// Handles events repository. Notice that the all methods should modify the Subject attribute of the
+    /// CloudEvent, since cosmosDb fails if Subject contains slashes '/'.
+    /// </summary>
+    public class EventsRepository : IEventsRepository
+    {
+        private readonly ILogger logger;
+        private readonly Uri databaseUri;
+        private readonly Uri collectionUri;
+        private readonly string collectionId = "events";
+        private readonly string partitionKey = "/subject";
+        private readonly string triggerId = "trgUpdateItemTimestamp";
+        private readonly string databaseId;
+        private static DocumentClient client;
+
```

Comment:
Same goes for the private variables here, begin with _

Reply:
Done :)